ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -797,16 +802,24 @@ define([
     //
     // Returns undefined if the latitude is out of range.
     //
-    UTMtoLL: function(utmParts) {
-      var results = converter.UTMtoLL(
-        utmParts.hemisphere === 'NORTHERN'
-          ? utmParts.northing
-          : utmParts.northing - northingOffset,
-        utmParts.easting,
-        utmParts.zoneNumber
-      )
+    utmUpstoLL: function(utmUpsParts) {
+      const { hemisphere, zoneNumber } = utmUpsParts
+      const northernHemisphere = hemisphere === 'NORTHERN' ? true : false
+
+      utmUpsParts = {
+        ...utmUpsParts,
+        northPole: northernHemisphere,
+      }
+      utmUpsParts.northing -=
```

Comment:
Here as well... The function should be handling all this for us.  The northingOffset variable should not need to exist